tests/e2e/test_actions.py

```diff
@@ -118,6 +120,8 @@ def test_run_playbook(update_environment):
         env=env,
     )
 
+    print(result.stdout)
```

Comment:
I think this is not needed.

Reply:
It was just while we troubleshoot the problem yesterday. It can be removed along with `debug=True`

Reply:
@Alex-Izquierdo I have removed it for now, but I am thinking we might need a way to enable debug output when we see errors on Travis, that doesn't happen in local Development environment, especially with race conditions.

Reply:
Travis? do you mean github action? If a fails occurs, pytest print the stacktrace and the values compared. If you find something that needs to be printed, I think it's better using the logger.